homeassistant/components/squeezebox/browse_media.py

```diff
@@ -78,83 +94,209 @@
     "Playlists": MediaType.PLAYLIST,
     "Genres": MediaType.GENRE,
     "Favorites": None,  # can only be determined after inspecting the item
+    "Apps": MediaClass.APP,
+    "Radios": MediaClass.APP,
+    "App": None,  # can only be determined after inspecting the item
     "New Music": MediaType.ALBUM,
+    MediaType.APPS: MediaType.APP,
+    MediaType.APP: MediaType.TRACK,
 }
 
 
+@dataclass
+class Squeezebox_maps:
+    """Class for browser to squeezebox mappings."""
+
+    content_type_to_child_type: dict[
+        str | MediaType,
+        str | MediaType | None,
+    ]
+    content_type_media_class: dict[str | MediaType, dict[str, MediaClass | None]]
+    squeezebox_id_by_type: dict[str | MediaType, str]
+    media_type_to_squeezebox: dict[str | MediaType, str]
+
+
 async def build_item_response(
     entity: MediaPlayerEntity,
     player: Player,
     payload: dict[str, str | None],
     browse_limit: int,
+    known_apps: set[str],
+    squeezebox_maps: Squeezebox_maps,
 ) -> BrowseMedia:
     """Create response payload for search described by payload."""
 
+    def update_squeezebox_maps(cmd: str | MediaType, type: str) -> None:
```

Comment:
Two things: This name should be more specific -- it's about adding a command. Second, don't nest the functions inside `build_item_response`. The cyclomatic complexity remains high when nested because of tracking the variable scope used in/out of the functions. Note, when moving the function to the module level, it would be appropriate to add an underscore to denote it's non-public outside this module.